Finmer.Game/Views/Parts/ItemTooltipView.xaml.cs

```diff
@@ -216,6 +218,10 @@ private static UIElement CreateCoinImage()
             return container;
         }
 
+        private void UserControl_DataContextChanged(object sender, DependencyPropertyChangedEventArgs e)
```

Comment:
This method is unused; please remove it

Reply:
not gonna lie, didn't even know this part of the code existed...

Reply:
> 
> 
> not gonna lie, didn't even know this part of the code existed...

Be sure to diff your changes (either `git diff` or right-click > 'Compare with Unmodified' in Visual Studio). A quick check over your changes before committing / pushing can help you with finding obvious problems and making sure the change looks sane.